.github/workflows/ci-memory-badger.yml

```diff
@@ -1,20 +0,0 @@
-name: CIT Memory And Badger
```

Comment:
makes sense.

Reply:
Actually, one reason why I wanted to have build tags was so that memory/badger storage tests would not need to run when we run ES/Cassandra tests. But they still seem to run per https://github.com/jaegertracing/jaeger/runs/4048978413?check_suite_focus=true

Reply:
I agree they should not run. Let me check

Reply:
I simply did not use a build-tag for those tests. 

Please let me know if you prefer to have a unique tag for all the 3 tests (grpc/badge/memory) instead of one for each

Reply:
As promissed: 

https://github.com/jaegertracing/jaeger/runs/4050387504?check_suite_focus=true